CRM/Financial/BAO/Order.php

```diff
@@ -0,0 +1,280 @@
+<?php
+/*
+ +--------------------------------------------------------------------+
+ | Copyright CiviCRM LLC. All rights reserved.                        |
+ |                                                                    |
+ | This work is published under the GNU AGPLv3 license with some      |
+ | permitted exceptions and without any warranty. For full license    |
+ | and copyright information, see https://civicrm.org/licensing       |
+ +--------------------------------------------------------------------+
+ */
+
+/**
+ *
+ * @package CRM
+ * @copyright CiviCRM LLC https://civicrm.org/licensing
+ *
+ * Order class.
+ *
+ * This class is intended to become the object to manage orders, including via Order.api.
+ *
+ * As of writing it is in the process of having appropriate functions built up.
+ */
+class CRM_Financial_BAO_Order {
+
+  /**
+   * Price set id.
+   *
+   * @var int
+   */
+  protected $priceSetID;
+
+  /**
+   * Selected price items in the format we see in forms.
+   *
+   * ie.
+   * [price_3 => 4, price_10 => 7]
+   * is equivalent to 'option_value 4 for radio price field 4 and
```

Comment:
I think this should read `for radio price field 3`

Reply:
Thanks - fixed!